nextflow.config

```diff
@@ -63,7 +65,9 @@ params {
     // Bismark default is 0.2 (L,0,-0.2), Bowtie2 default is 0.6 (L,0,-0.6)
     meth_cutoff                = null
     no_overlap                 = true
+    ignore                     = 0
```

Comment:
I don't like calling this parameter `ignore`, that seems far too generic a name.